pkg/middleware/middleware.go

```diff
@@ -29,12 +30,18 @@ func AddDefaultResponseHeaders(cfg *setting.Cfg) web.Handler {
 	t := web.NewTree()
 	t.Add("/api/datasources/uid/:uid/resources/*", nil)
 	t.Add("/api/datasources/:id/resources/*", nil)
+
 	return func(c *web.Context) {
 		c.Resp.Before(func(w web.ResponseWriter) { // if response has already been written, skip.
 			if w.Written() {
 				return
 			}
 
+			traceId := tracing.TraceIDFromContext(c.Req.Context(), false)
+			if traceId != "" {
+				w.Header().Set("traceparent", traceId)
+			}
```

Comment:
This doesn't seem to follow the format given by [w3c/trace-context](https://www.w3.org/TR/trace-context-1/#traceparent-header-field-values)?

Reply:
@sakjur sorry, you are correct, i will fix this (either by giving it a new name or by making it the right format)

Reply:
adjusted it to be "grafana-trace-id" (i can change it even more if needed, no problem)